boards/arm/gd32e507v_start/doc/index.rst

```diff
@@ -81,12 +81,13 @@ follows:
 
 - JP3/4: Select 2-3 for both (boot from user memory)
 
-Using J-Link
-=============
+Using GD-Link or J-Link
+=======================
 
-While the board comes with an embedded GD-Link programmer, it is not yet
-supported by OpenOCD. However, J-Link can be used to program the board using
-the SWD interface exposed in the JP1 header.
+The board comes with an embedded GD-Link programmer, however, it is only
+supported when using pyOCD, not OpenOCD. Before programming, make sure you
+have the ``GigaDevice.GD32E50x_DFP`` CMSIS-Pack in pyOCD. J-Link can also be
+used to program the board using the SWD interface exposed in the JP1 header.
```

Comment:
I think it is friendly that describe here how to install CMSIS-Pack.

(I described in #49085  this PR as a reference.)
https://github.com/zephyrproject-rtos/zephyr/pull/49085/files#diff-9453a38bf6cfa4fbad48ba5eec5a37872638d200d86b2fcee0d212df964e81f6R86-R94

Reply:
done